flink-ml-benchmark/src/main/resources/stringindexer-benchmark.json

```diff
@@ -17,7 +17,7 @@
   "version": 1,
   "stringindexer100000000": {
     "inputData": {
-      "className": "org.apache.flink.ml.benchmark.datagenerator.common.RandStringGenerator",
+      "className": "org.apache.flink.ml.benchmark.datagenerator.common.RandomStringGenerator",
```

Comment:
Let's add a test case to verify that all existing benchmark json files are valid. This means the class names correspond to existing classes, and that the param maps are reasonable.